defaults/main.yml

```diff
@@ -58,6 +58,8 @@ nimbus_portal_client_rpc_enabled: true
 nimbus_portal_client_rpc_address: '127.0.0.1'
 nimbus_portal_client_rpc_port: 9900
 nimbus_portal_client_rpc_api_options: 'portal,discovery'
+nimbus_portal_client_web3_enabled: false
+nimbus_portal_client_web3_url: 'http://127.0.0.1:8545'
```

Comment:
But we don't really know that the port would be `8545`, so the more correct approach would leave this unset and make it mandatory in the template. That way you force the user to set it explicitly.